starknet_devnet/starknet_wrapper.py

```diff
@@ -135,13 +143,62 @@ async def initialize(self):
             await self.fee_token.deploy()
             await self.accounts.deploy()
             await self.__deploy_chargeable_account()
-            await self.__predeclare_oz_account()
+            await self.__predeclare_starknet_cli_account()
             await self.__udc.deploy()
 
             await self.__preserve_current_state(starknet.state.state)
-            await self.create_empty_block()
+            await self.__create_genesis_block()
             self.__initialized = True
 
+    async def __create_genesis_block(self):
+        """Create genesis block"""
+        transactions: List[DevnetTransaction] = []
+        transaction_hash = 1
+
+        # Declare transactions
+        declare_hashes = [
+            to_bytes(FeeToken.HASH),
+            to_bytes(UDC.HASH),
+            ChargeableAccount(self).class_hash_bytes,
+            to_bytes(STARKNET_CLI_ACCOUNT_CLASS_HASH),
+        ]
+        for class_hash in declare_hashes:
+            internal_declare = create_empty_internal_declare(transaction_hash, class_hash)
+            declare_transaction = create_genesis_block_transaction(
+                internal_declare, TransactionType.DECLARE
+            )
+            transactions.append(declare_transaction)
+            transaction_hash += 1
+
+        # Deploy transactions
+        deploy_data = [
+            (to_bytes(FeeToken.HASH), FeeToken.ADDRESS),
+            (to_bytes(UDC.HASH), UDC.ADDRESS),
+            (ChargeableAccount(self).class_hash_bytes, ChargeableAccount(self).ADDRESS),
+        ]
+        for account in self.accounts:
+            deploy_data.append((account.class_hash_bytes, account.address))
+
+        for deploy_tuple in deploy_data:
```

Comment:
Can be replaced with
```python
for class_hash, contract_address in deploy_data:
    ...
```

Reply:
Cool! Thanks! Didn't know that! Fixed in 0cc7c1580550bf771d6fde40cadea7f539483583